packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -11,11 +11,11 @@ import {ChakraProvider} from '@chakra-ui/react'
 // Removes focus for non-keyboard interactions for the whole application
 import 'focus-visible/dist/focus-visible'
 
-import theme from '../../theme'
-import {MultiSiteProvider} from '../../contexts'
-import {resolveSiteFromUrl, resolveLocaleFromUrl} from '../../utils/site-utils'
+import theme from 'retail-react-app/app/theme'
```

Comment:
This change to pathing is widespread, I recommend filtering the changes via the sidebar here `pwa-kit-dev` and `my-extended-retail-app` are the bulk of meaningful changes

<img width="331" alt="image" src="https://user-images.githubusercontent.com/47546998/235266752-9cf503c5-1d47-491a-a541-6064ff03a907.png">